pages/design-studio.md

```diff
@@ -38,5 +38,5 @@ No PRA implications. If conducted with nine or fewer members of the public, the
 
 ## Additional resources
 
-A [presentation by Todd Zaki Warfel.](https://vimeo.com/37861987/)
+A [presentation by Todd Zaki Warfel](https://vimeo.com/37861987/)
```

Comment:
This is annoying because the rest of the words ended up on a separate line, so GH isn't showing that line in every view, but this link ends in the middle of a longer sentence fragment.